protocol-units/bridge/service/src/chains/ethereum/client.rs

```diff
@@ -650,11 +650,32 @@ impl GasMaster for EthClient {
 	}
 
 	async fn get_priority_gas_fee_estimate(&self) -> GasMasterResult<u64> {
```

Comment:
Perhaps we should use u128 for all gas and amount value because of the decimal use in Eth for example, you can exceed the u64 limit. I got the issue when calculating the transaction fee for Eth. The code: `let transaction_fee_wei = estimate_gas * gas_price;` has exceeded u64 during my test.